config/features.yaml

```diff
@@ -75,3 +75,14 @@ github_accounts:
         aggregate: True
         tags: ['Github Account']
         emojis: ['ID_BUTTON']
+
+# Linkedin account extraction from profile edit url
+linkedin_accounts:
+        query_string: 'source_short:"WEBHIST" AND
+          url:"https://www.linkedin.com/in/" AND "/edit/"'
+        attribute: 'url'
+        store_as: 'found_account'
+        re: 'https://www.linkedin.com/in/([A-z-\d]{1,39})/edit/'
```

Comment:
Looks like LinkedIn account names can be 5-30 characters long, with a language id at the end (unclear if this is in addition to 30 or included). Maybe change the re to {5,32} ?

https://www.linkedin.com/help/linkedin/answer/87/customizing-your-public-profile-url?lang=en

Reply:
Done